tests/stub/driver_execute_query/scripts/transaction_chaining_custom_bmm.script

```diff
@@ -0,0 +1,42 @@
+!: BOLT 5.0
+!: ALLOW CONCURRENT
+
+A: HELLO {"{}": "*"}
+*: RESET
+{*
+    {{
+        C: BEGIN {"bookmarks": ["other_db:bm1"], "[db]": "*"}
+        S: SUCCESS {}
+        {{
+            C: RUN "CREATE (p:Person{name:$name}) RETURN p.name AS name" { "name":"a person" } {}
+            S: SUCCESS {"fields": ["name"]}
+            C: PULL {"n": 1000}
+            S: RECORD ["a person"]
+               SUCCESS {"type": "w"}
+        ----
+            C: RUN "MATCH (p:Person{name:$name}) RETURN p.name AS name" { "name":"a person" } {}
+            S: SUCCESS {"fields": ["name"]}
+            C: PULL {"n": 1000}
+            S: SUCCESS {"type": "r"}
+        }}
+        C: COMMIT
+        S: SUCCESS {"bookmark": "bm1"}
+    ----
+        {{
+            C: BEGIN {"bookmarks": ["bm1", "other_db:bm1"], "[db]": "*"}
+        ----
+            C: BEGIN {"bookmarks": ["other_db:bm1", "bm1"], "[db]": "*"}
+        }}
```

Comment:
```suggestion
        C: BEGIN {"bookmarks{}": ["bm1", "other_db:bm1"], "[db]": "*"}
```

Reply:
I don't want to force the order in the bookmark list.

Reply:
For example, js sends `"other_db:bm1", "bm1"`

Reply:
I know. That's what the `{}` after the key name is there for. See [Stub script specs (internal doc)](https://docs.google.com/document/d/1QMIVtOTTuNEEZFAVumWGM-BWhLY76wJ56rIAQAisXtA/edit?pli=1#heading=h.horfqne0jrha).